<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CustomField" name="firstName" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>500</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Зареєструватися</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="passwordVisibility">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomField" name="password" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>500</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;РЕЄСТРАЦІЯ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="CustomField" name="thirdName" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>500</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomField" name="secondName" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>500</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomField" name="email" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>500</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="birthdayCalendar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>256</width>
     <height>163</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Дата народження&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>470</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Повернутись</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="maleRadioButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Чоловіча</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="femaleRadioButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Жіноча</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Стать при народженні&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomField</class>
   <extends>QWidget</extends>
   <header location="global">Authorization/CustomField/CustomField.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
